src/numbers/RequestErrorDetector.java

```diff
@@ -0,0 +1,91 @@
+package numbers;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+class RequestErrorDetector {
```

Comment:
This class is unreadable, please refactor else/ifs and extract complex parts into methods

Reply:
Got it